### PR TITLE
Executed commands script

### PR DESCRIPTION
**Pull request by gshriki**
Script to list all executed commands in war room - by playbook, by user, or both

@NeelRust FYI

### CODE REVIEW

File: Scripts/script-listExecutedCommands.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+commonfields:
+  id: listExecutedCommands
+  version: -1
+name: listExecutedCommands
+script: |-
+  var fixNewLines = function(str) {
+      var tmp = str.split('\n').join('<br>');
```
Comment (dorsha) [inline Scripts/script-listExecutedCommands.yml]: You can just do 
`return str.split('\n').join('<br>').split('\r').join(' ');`

Also, why not using replace?

File: Scripts/script-listExecutedCommands.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  }
+  if (userCommands.length > 0) {
+      md += tableToMarkdown('User Executed Commands', userCommands, ['Time', 'Entry ID', 'User', 'Command']) + '\n';
+  }
+  if (md === '') {
+      md = 'No commands found\n';
+  }
+  return {ContentsFormat: formats.markdown, Type: entryTypes.note, Contents: md};
+type: javascript
+tags: []
```
Comment (dorsha) [inline Scripts/script-listExecutedCommands.yml]: Do you want to tag it?

File: Scripts/script-listExecutedCommands.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+comment: Lists  executed commands in War Room
+enabled: true
+args:
+- name: source
```
Comment (dorsha) [inline Scripts/script-listExecutedCommands.yml]: missing description for source

File: TestPlaybooks/playbook-listExecutedCommands-Test.yml [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+      scriptName: AreValuesEqual
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "no":
+      - "8"
+    scriptarguments:
+      left: ${withData=val.trim()}
+      right: No commands found
```
Comment (orius123) [inline TestPlaybooks/playbook-listExecutedCommands-Test.yml]: Shouldn't it find results in this case (The playbook `Print` task)?

Comment (gshriki) [reply TestPlaybooks/playbook-listExecutedCommands-Test.yml]: So first time I'm calling it in the playbook, i use the source=manual, so it shouldn't find. The second time I run it, I use source=playbook. and the playbook checks those conditions.

Comment (orius123) [reply TestPlaybooks/playbook-listExecutedCommands-Test.yml]: and you navigate only on the "no" condition. saw it just now.

**Comment by dorsha** [comment]
@gshriki please resolve conflict, and merge or handle this PR.

**Comment by dorsha** [comment]
@gshriki please merge from master and merge

**Pull request merged**